packages/authgear-react-native/android/src/main/java/com/authgear/reactnative/AuthgearReactNativeModule.java

```diff
@@ -972,15 +972,15 @@ public void signWithDPoPPrivateKey(ReadableMap options, Promise promise) {
     public void checkDPoPPrivateKey(ReadableMap options, Promise promise) {
         String kid = options.getString("kid");
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
-            promise.resolve(true);
+            promise.resolve("true");
             return;
         }
         String alias = this.formatDPoPKeyAlias(kid);
         try {
             this.getPrivateKey(alias);
-            promise.resolve(true);
+            promise.resolve("true");
         } catch (Exception e) {
-            promise.resolve(false);
+            promise.resolve("false");
```

Comment:
Why?

Reply:
https://github.com/authgear/authgear-sdk-js/blob/master/packages/authgear-react-native/src/nativemodule.ts#L5-L7

Reply:
But that is about native reading data from JS. So JS reading data from native has this problem as well?